src/index.js

```diff
@@ -0,0 +1,27 @@
+// import _ from 'lodash';
+import './style.css';
+
+const tasks = [
+  {
+    description: 'cleaning',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'accounting',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'cooking',
+    completed: false,
+    index: 2,
+  },
+];
+
+document.querySelector('.mylist').innerHTML = tasks.map((task) => `<li>
+    <div class="task-container">
+    <input type="checkbox" id="checkbox${task.index}">  
+    <h4>${task.description}</h4>
+    </div>
+  </li>`).join('');
```

Comment:
- [ ] Please style your to-do-list according to the specification given. I have attached an image showing you everything that is missing in your design. Please implement it as it is required to progress to the next milestone 👍 
### Design Specs
![image](https://user-images.githubusercontent.com/46839548/183965864-4f6ee398-6181-4eb0-a265-1be09bf8a168.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/183965946-a13c0b52-78ab-46dc-ad4a-c5f4d5500243.png)